### ISSUE TITLE
[BUG]: UserID is int64, not int32

### ISSUE THREAD

**Issue by r1der23** [xelaj/mtproto#133]
Unfortunately, when i have tried to send message to bot, i found that bot id is more than int32)
etc

**Comment by r1der23** [comment]
btw sorry for my english and code quality, i m newbie

**Comment by KovshefulCoder** [comment]
Yes, the same problem, telegram does not accepting current AppID, so without 133 layer support by this package you won`t be able to even authenticate telegram account

**Comment by KovshefulCoder** [comment]
@r1der23 links are unavailable

**Comment by vench** [comment]
Hello. Thanks for you job. 
@r1der23  - update please links or share please your code
